### DISCUSSION TITLE
Traces information from Vector to AWS x-ray

### DISCUSSION THREAD

**Discussion by MichaelShapira** [vectordotdev/vector#18862] [Q&A]
Hello.
I see that Vector supports AWS CloudWatch logs and metrics. Does X-ray traces are being send as part of this data?
If not what is the solution to send the traces to AWS X-ray?

**Comment by jszwedko** [comment]
Vector doesn't yet support sending traces to AWS X-Ray. Would you want to open a feature request?

**Reply by MichaelShapira** [reply]
Yes. I see that there is already implementation for CloudWatch metrics and logs. To close the loop for all 3 moving parts (metrics, logs, traces) it should support also X-ray (for traces)
